modules/nw-disabling-hsts.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-disabling-hsts_{context}"]
+= Disabling HTTP strict transport security per-route
+
+To disable HTTP strict transport security (HSTS) per-route, you can set the `max-age` value in the route annotation to `0`.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You installed the `oc` CLI.
+
+.Procedure
+
+* To disable HSTS, set the `max-age` value in the route annotation to `0`, by entering the `oc annotate` command:
++
+[source,terminal]
+----
+$ oc annotate route routename -n namespacename --overwrite=true "haproxy.router.openshift.io/hsts_header"="max-age=0"
+----
```

Comment:
We could add even more with a description of how to disable it for every route in a namespace:
`$ oc annotate route --all -n namespacename --overwrite=true "haproxy.router.openshift.io/hsts_header"="max-age=0"`

And how to query the annotation for all routes:
`$ oc get route  --all-namespaces -o go-template='{{range .items}}{{if .metadata.annotations}}{{$a := index .metadata.annotations "haproxy.router.openshift.io/hsts_header"}}{{$n := .metadata.name}}{{with $a}}Name: {{$n}} HSTS: {{$a}}{{"\n"}}{{else}}{{""}}{{end}}{{end}}{{end}}'`

Name: routename HSTS: max-age=0